### PR TITLE
add aws glue server support

### PR DESCRIPTION
**Pull request by saugerDecathlon**
#46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -341,10 +341,10 @@ This object _MAY_ be extended with [Specification Extensions](#specification-ext
...
-| description | `string` | An optional string describing the server.                                                                                                                                                               |
+| Field       | Type     | Description                                                                                                                                                                                                               |
+|-------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| type        | `string` | REQUIRED. The type of the data product technology that implements the data contract. Well-known server types are: `bigquery`, `s3`, `redshift`, `snowflake`, `databricks`, `postgres`, `kafka`, `pubsub`, `local`, `glue` |
```
Comment (simonharrer) [inline README.md:346]: local should always be the last type.

File: README.md [hunk 2]
```diff
@@ -439,6 +439,27 @@ servers:
...
 | format | `string` | The format of the file(s), such as `parquet`, `csv`, or `json`.                     |
 
+#### AWS Glue Server Object
+
+| Field    | Type     | Description                                                |
+|----------|----------|------------------------------------------------------------|
+| type     | `string` | `glue`                                                     |
+| account  | `string` | The AWS account, e.g., `1234-5678-9012`                    |
+| database | `string` | The AWS Glue Catalog database                              |
+| location | `string` | S3 path, starting with `s3://`                             |
```
Comment (simonharrer) [inline README.md:449]: why is the location on s3 relevant here?

File: README.md (hunk 1 above)
Comment (saugerDecathlon) [reply README.md]: updated

File: README.md [hunk 3]
```diff
@@ -431,6 +431,29 @@ servers:
...
 | topic   | `string` | The topic name.       |
 
+
+#### AWS Glue Server Object
+
+| Field    | Type     | Description                                                |
+|----------|----------|------------------------------------------------------------|
+| type     | `string` | `glue`                                                     |
+| account  | `string` | The AWS account, e.g., `1234-5678-9012`                    |
+| database | `string` | The AWS Glue Catalog database                              |
```
Comment (simonharrer) [inline README.md:441]: Required fields should be marked REQUIRED at the beginning of the description

Comment (saugerDecathlon) [reply README.md]: updated

**Pull request merged**